coalib/bears/requirements/PythonRequirement.py

```diff
@@ -83,6 +103,15 @@ def multiple(cls, *args):
                     reqs.append(cls(name, version),)
                 except ValueError:
                     raise TypeError('The tuple must have 2 elements.')
+            elif isinstance(requirement, list):
+                if len(requirement) == 1:
+                    reqs.append(cls(requirement),)
+                else:
+                    try:
+                        name, version = requirement
+                        reqs.append(cls(name, version),)
+                    except ValueError:
+                        raise TypeError('The list must have max 2 elements.')
```

Comment:
nonononono this is code duplication!

Just remove the whole isinstance checking, it's not pythonic. Just invoke the len and do the pattern matching and if that fails make a TypeError that you need any iterable nongenerator with twoish elements.